README.md

```diff
@@ -120,9 +120,9 @@ Example:
 
 ```js
 var fsBlobStore = require('fs-blob-store')  // an in-memory blob store
```

Comment:
true, should had tested it. Fixed it now https://github.com/ipfs/js-ipfs-repo/commit/c2c76d7df55381e89f3a3558b9f98962e3a1dd46